config/routes.rb

```diff
@@ -1,5 +1,16 @@
 Rails.application.routes.draw do
+ HEAD
+ HEAD
+  resources :users
+end
+
+  # resourcesを使用したルーティング
+  resources  :tasks
+end
+master
+
   resources :messages
   #resourceを使用したルーティング
   resources :tasks
 end
+master
```

Comment:
`>>>` だけでなく，その行に含まれれる `HEAD`, `master` などももちろん削除しなければなりません。

```suggestion
Rails.application.routes.draw do
  resources :users
  resources :messages
  resources :tasks
end
```